---
{"zotero-key":"T5P5SDRD","zt-attachments":["21199"],"alias":", @LFS252M09 Tutorial Questions","keywords":[],"FirstAuthor":"[[ ]]","dg-publish":true,"permalink":"/sources/lfs-252-m09-tutorial-questions/","dgPassFrontmatter":true}
---

# @LFS252M09TutorialQuestions

>[!Synth]  
>**Contribution**::  
>  
>**Related**:: 
>  

> [!Abstract]+
> 

> [!Info]
> title: LFS252M09 Tutorial Questions
> type:: 
> date:: 
> DOI:: 
> FirstAuthor:: 
> creators:: 

> [!Links]+
>  | Incoming Links                                                        |
> | --------------------------------------------------------------------- |
> | [[Sources/LFS252M09TutorialQuestions\|LFS252M09TutorialQuestions]] |
> 
{ .block-language-dataview}


[Zotero](zotero://select/library/items/T5P5SDRD) [attachment](<file:///Users/nathanmaxwell/Zotero/storage/HCKPQLF8/Molecular%20Biology%20Tutorial%20Week%209.pdf>)

> [!quote-blue] (Page 2) blue
> 
> What is a secondary and tertiary RNA structure? How are these structures formed?
> 
> ---
> 2nd RNA strutures and based on complementary based paired hydrogen bonds. 3rd RNA structures and more complex the 2nd with electrostatic forces (Van wals forces)
>
{ #BEA4XLAFaHCKPQLF8p2}


> [!quote-blue] (Page 2) blue
> 
> What are some uses of these structured RNAs in eukaryotic cells?
> 
> ---
> 2nd RNA uses include gene regulation and protein translation. 3rd RNA can catalyze biochemical reactions
>
{ #VX5HM7NVaHCKPQLF8p2}


> [!quote-blue] (Page 5) blue
> 
> Describe what a ribozyme is, and describe one function
> 
> ---
> They are catabolically active RNA molecules
>
{ #DMAENTYRaHCKPQLF8p5}


> [!quote-blue] (Page 6) blue
> 
> What does the term RNA interference (RNAi) refer to? What is the primary roles of these mechanisms?
> 
> ---
> Post-transcriptional gene silencing
>
{ #ZTTDEEDBaHCKPQLF8p6}


> [!quote-blue] (Page 6) blue
> 
> Describe the steps in the miRNA pathway
> 
> ---
> 1) miRNA forms hairpin loop
> 2) Drosha cuts off non complementary regions
> 3) Dicer slices the miRNA into 20-23bp chains
> 4) formation of the RISC complex
>
{ #7U6GD439aHCKPQLF8p6}


> [!quote-blue] (Page 7) blue
> 
> Define the term biomarker
> 
> ---
> A biomarker is a measurable indicator of a biological process
>
{ #IYADH5G6aHCKPQLF8p7}


> [!quote-blue] (Page 7) blue
> 
> Why are miRNAs good biomarkers?
> 
> ---
> stability and specificity as miRNAs are highly specific to particular tissues
>
{ #LTXWGEJPaHCKPQLF8p7}


> [!quote-blue] (Page 9) blue
> 
> Describe the steps of the siRNA pathway
>
{ #ZYJLALRTaHCKPQLF8p9}


> [!quote-blue] (Page 10) blue
> 
> A virus attacks a cell, and produces dsRNA during its  replication, which RNAi pathway is most likely to be  activated?
> 
> ---
> siRNA pathway
>
{ #L965YHGIaHCKPQLF8p10}


> [!quote-blue] (Page 10) blue
> 
> An RNA molecule from the host genome is  transcribed and folds into a dsRNA structure, which  RNAi pathway is most likely to be activated?
> 
> ---
> miRNA
>
{ #YUNJ9SMSaHCKPQLF8p10}


> [!quote-blue] (Page 10) blue
> 
> A scientist injects dsRNA they have synthesized,  which RNAi pathway is most likely to be activated?
> 
> ---
> siRNA
>
{ #9T86BKRFaHCKPQLF8p10}


> [!quote-blue] (Page 11) blue
> 
> Describe two similarities between the miRNA and siRNA pathways
> 
> ---
> Both pathways involve the dicer and RISC complex
>
{ #684P5KQFaHCKPQLF8p11}


> [!quote-blue] (Page 11) blue
> 
> Describe two differences between the miRNA and siRNA pathways
> 
> ---
> miRNAs are endogenously expressed by the host genome. In contrast, siRNAs are typically introduced into the cell through transfection or other means
>
{ #PKFXTRHSaHCKPQLF8p11}


> [!quote-blue] (Page 11) blue
> 
> Where can the miRNA and siRNA pathways be utilized for scientists, eukaryotes, prokaryotes, both, or neither
> 
> ---
> The miRNA and siRNA pathways can be utilized inÂ <b>eukaryotes</b>.
>
{ #3A8G7WAMaHCKPQLF8p11}


> [!quote-blue] (Page 14) blue
> 
> What is the purpose of the Shine-Dalgarno sequence in prokaryotic translation?
> 
> ---
> A Shine-Dalgarno sequence marks the start of each coding sequence, letting the ribosome find the right start codon for each gene.
>
{ #F3PJLQEGaHCKPQLF8p14}


> [!quote-blue] (Page 15) blue
> 
> Describe the formation of the 30S initiation complex in two steps, use a diagram if this helps.
> 
> ---
> 1) IF-3 & IF-1, start codon on the P-site
> 2) the anticodons bind to the codons
>
{ #S7WIQ6B3aHCKPQLF8p15}


> [!quote-blue] (Page 16) blue
> 
> What is the final complex in translation initiation called?
> 
> ---
> Initiation complex
>
{ #33DXU4XRaHCKPQLF8p16}


> [!quote-blue] (Page 16) blue
> 
> In which site of the ribosome is the fMet-tRNA now located?
> 
> ---
> P-section
>
{ #IQIACY4GaHCKPQLF8p16}


> [!quote-blue] (Page 17) blue
> 
> What are the three steps in elongation?
> 
> ---
> binding of the tRNA to the ribosome, bonding of the peptides, translocation of the ribosome
>
{ #XBCWAI5XaHCKPQLF8p17}


> [!quote-blue] (Page 17) blue
> 
> Name the two components of the complex required for the next tRNA to enter the A site.
> 
> ---
> Aminoacyl-tRNA, elongation factor
>
{ #PMWTS6DWaHCKPQLF8p17}


> [!quote-blue] (Page 17) blue
> 
> Name the two components of the complex required for the next tRNA to enter the A site.
>
{ #NGTQ9MPWaHCKPQLF8p17}


> [!quote-blue] (Page 17) blue
> 
> What  happens to these components of the complex once the tRNA is in position?
> 
> ---
> they get released to be recycled
>
{ #3HPQ7RRFaHCKPQLF8p17}


> [!quote-blue] (Page 19) blue
> 
> What type of molecule is peptidyl transferase?
> 
> ---
> an RNA enzyme
>
{ #J5PMQZZEaHCKPQLF8p19}


> [!quote-blue] (Page 20) blue
> 
> What molecular mechanisms cause the movement of the ribosome along the mRNA strand? Why is GTP hydrolysis needed in the elongation process?
> 
> ---
> GTP releases energy needed for the ribosome to move long the mRNA strand
>
{ #U8TG8UIXaHCKPQLF8p20}


> [!quote-blue] (Page 21) blue
> 
> What signals termination of translation to occur and describe what happens next to terminate the translation process.
> 
> ---
> release factors cleave the peptide chain, releasing it from the ribosome.
>
{ #4MKGJ3DJaHCKPQLF8p21}


> [!quote-blue] (Page 22) blue
> 
> List 4 key components required for translation
> 
> ---
> mRNA, Initiation factors, elongation factors, GTP
>
{ #YKGDRFCAaHCKPQLF8p22}


> [!quote-blue] (Page 22) blue
> 
> List 2 major differences between prokaryote and eukaryote protein translation.
>
{ #DSHGGRNSaHCKPQLF8p22}


> [!quote-blue] (Page 22) blue
> 
> The large ribosome subunit has 3 sites: A, P and E. In which site will the polypeptide chain be present during translation?
> 
> ---
> P site
>
{ #ZRM9D66PaHCKPQLF8p22}


> [!quote-blue] (Page 22) blue
> 
> The translation termination is encoded by a codon on the mRNA that correlates to which amino acid on the tRNA?
> 
> ---
> Stop codon
>
{ #BCFZ6ZKYaHCKPQLF8p22}





